pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -405,18 +410,13 @@ func (i *KumaInjector) NewAnnotations(pod *kube_core.Pod, mesh *mesh_core.MeshRe
 		annotations[metadata.CNCFNetworkAnnotation] = metadata.KumaCNI
 	}
 
-	// By default always set 'kuma.io/virtual-probes: enabled' if it's not set yet.
-	if _, exist, _ := metadata.Annotations(pod.Annotations).GetEnabled(metadata.KumaVirtualProbesAnnotation); !exist {
-		annotations[metadata.KumaVirtualProbesAnnotation] = metadata.AnnotationEnabled
+	if err := virtualProbesEnabled(annotations, pod, i.cfg); err != nil {
+		return nil, err
```

Comment:
please change the method names. I was confused why it's not returning bool and what is going on. It should be something like `setVirtualProbesEnabledAnnotation`, `setVirtualProbesPortAnnotation`